eahub/profiles/management/commands/profilesfromlegacyhub.py

```diff
@@ -19,6 +19,8 @@ def collect_cause_areas(
     cause_prioritisation,
     meta,
     climate_change,
+    mental_health,
```

Comment:
Revert this; as written it'll cause a runtime error, and we're not planning on re-running this import anyway, I think.